specs/features/Inconsistencies.md

```diff
@@ -0,0 +1,126 @@
+# Inconsistencies in Adaptive Cards platforms
+
+## Motivation
+
+Adaptive Cards is currently available on these platforms:
+
+* [UWP](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/uwp/getting-started) (Available in all WinRT-projected languages; renders using XAML)
+* .Net (Available in all .Net languages; renders using [HTML](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/net-html/getting-started) or [WPF](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/net-wpf/getting-started))
+* [Android](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/android/getting-started) (Java; renders using native Android platform)
+* [iOS](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/ios/getting-started) (ObjC; renders using native iOS platform)
+* [Javascript](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/javascript/getting-started) (renders using HTML/JS/CSS)
+* [ReactNative](https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/react-native/getting-started) (community renderer)
+* Flutter (community renderer)
+
+This list of platforms represents at least 7 distinct implementations of Adaptive Cards at the rendering level. At each decision point along the way during implementation, the Adaptive Cards developer (i.e. someone writing code to parse or render Adaptive Cards) risks implementing behavior that diverges from intended behavior. One of the biggest selling points for Adaptive Cards is our ability to render across our supported platforms in a way that at once feels native to the supported platform while also having a look and feel that's representative of the card author's intent. Indeed, portability is one of our [core goals](https://docs.microsoft.com/en-us/adaptive-cards/#goals). Each time an inconsistency sneaks into the platform, it risks introducing costs to our partners. These costs have real consequences for the adoption of Adaptive Cards in the market:
+2
+* Existing customers are less likely to adopt new versions of Adaptive Cards, even if they want the newer features
+* Potential new customers might be scared to adopt Adaptive Cards altogether if there's a consistency or stability problem (perceived or real), since cross-platform capabilities are central to our value proposition
+* When existing customers *do* adopt new versions, they may end up having to spend extra time testing new versions and then deal with new bugs when *their* customers complain
+
+### In order to maintain our pledge of cross-platform capability, and in order to support all of our partners and customers, we **must** take a strong stance against the introduction of inconsistencies in Adaptive Cards
```

Comment:
> pledge of cross-platform capabilit [](start = 28, length = 35)

Elaborate a bit more to be be specific and include pointers to actual public docs or some such here. Also "pledge" seems a bit dramatic :) - Can stick to "goals" or vision as outlined [add link]; In addition customers, esp those spanning product offerings spanning multiple platforms have all reported numerous bugs and pain points esp as they upgrade or expand adaptive cards usage across these platforms (link to the github issues here reported by teams/cortana/teams desktop/bot framework)